src/main/scripts/test-haskell-backend

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -ueo pipefail
+
+rm -rf haskell-backend-kore
+git clone 'https://github.com/kframework/kore' haskell-backend-kore
```

Comment:
this doesn't seem at all the right way to do this. you should simply use the source directory of the haskell backend that has already been checked out.